src/libraries/System.Net.Http/tests/StressTests/HttpStress/build-local.ps1

```diff
@@ -14,7 +14,7 @@ if (-not ([string]::IsNullOrEmpty($args[0]))) {
 
 $LibrariesConfiguration = "Release"
 if (-not ([string]::IsNullOrEmpty($args[1]))) {
-    $LibrariesConfiguration = $args[0]
+    $LibrariesConfiguration = $args[1]
```

Comment:
Fixes in this file are unrelated. Both `$StressConfiguration` and `$LibrariesConfiguration` referred to `$args[0]` 🤦 .